airflow.cfg

```diff
@@ -278,7 +295,7 @@ worker_refresh_interval = 30
 
 # Secret key used to run your flask app
 # It should be as random as possible
-secret_key = temporary_key
+secret_key = 28d1c8bda62bb47e67d935c936e301a9f5ddaf40a5de875cf6b7765811b3
```

Comment:
And here?

Reply:
Similar to the Fernet key, I've added a [Flask secret key environment variable](https://airflow.apache.org/docs/apache-airflow/stable/configurations-ref.html?highlight=configuration#config-webserver-secret-key). 

The docs list three environment variables but it seems like you would only have to set one?